core/primitives-core/src/config.rs

```diff
@@ -86,18 +86,16 @@ pub struct VMLimitConfig {
     pub max_functions_number_per_contract: Option<u64>,
 }
 
-impl Default for VMConfig {
-    fn default() -> VMConfig {
+impl VMConfig {
+    pub fn test() -> VMConfig {
```

Comment:
shouldn't we mark it as #[cfg(feature = "test_features")] ?

Reply:
I think not, at least not initially. Currently, we use `test_features` only for things which are actively harmful to keep in the production binary (adversarial features, extra fields in data structures, that stuff). "append only" test code is generally not gated. 

We *could* gate all test code, but then means that `cargo test` would be useless, as it won't actually run any tests, and you'd have to pass `cargo test --features test_features` to run them. What's worse, the test would run *only* against the code with extra test functionality compiled in, which would be different from production code.